### ISSUE TITLE
Public Responses

### ISSUE THREAD

**Issue by tsgsjeremy** [jonstokes/shootercontrol#4]
**NOTE:** I'm an I.T. guy, so my focus is more towards the technology required to implement the stated goals and structure. That doesn't mean I won't give my 2 cents regarding said goals and structure. :)

----Original Post Follows----

Use this issue to leave suggestions for the project. NOTE: Read existing responses first to prevent duplicates. If you find your suggestion, up-vote the post.

These are some of my suggestions:
- **Privacy** - In order to make this work, members will necessarily have to share private information such as type and number of guns owned. Therefore we must do everything we can to prevent the data from becoming a "National Registry" used by and for the government.
  a) NO db data, whether containing personal info or not, should be provided to government agencies without a subpoena. 
  b) Access to live data must be restricted. At least 2 individuals but less than 10. Open to suggestions for how these folks should be selected.
- **Structure** - The structure should be similar to the secure certificate trust model, and be honored like a carry license (after verification)
  a) Start of Authority should be a board elected by the membership and serve for 5 years
  b) SOA would be responsible for vetting and authorizing intermediate authorities - either private or public. Gun clubs, range operators, NRA, police departments, FFLs, etc.
  c) Individual gun owners would be vetted and issued Certificates/Licenses by a trusted authority.
- **Enforcement** - to my mind, this is the key to making the whole thing work. The goal is to have both top down and bottom up accountability.
  a) Punishment for violators must be graduated and equal to the offense. Minor offenses may be subject to fines or temporary revocation of trust. And serious offenses may lead to permanent loss of trust for both the authority as well as any members they may have certified.
  b) Violations of law by members would result in punishment for the member and the immediate certifying authority.
  c) Multiple violations by a certifying authority would result in punishment for that authority and it's immediate certifying authority.
- **Possible process for certifying authority (CA) dealing with individual (IND)** (process would be different for certifying a group)
  a) CA runs all legally required background checks such as NICS.
  b) CA runs private background check (Ideally the COA would have a contract with a reputable background checking firm)
  c) IND supplies references to be checked by CA (specifics are a bit fuzzy)
  d) Process should take no more than 7-10 business days

That's all I have time for right now. Please critique and make your own suggestions.

**Comment by jonstokes** [comment]
@tsgsjeremy thanks for this! I just put up my thing, but will take a look at this in a bit and give more feedback. I've been kind of swamped with the response to the Medium (and now Vox) stuff, so it's taking me a while.

**Comment by nalanbar** [comment]
I like the idea, but without CCW reciprocity, it loses much of it's usefulness for most people who use their firearms as defensive tools. If your network thinks you are trustworthy enough to own a gun, and the government agrees, then you should also be able to use your network trust for CCW purposes.

**Comment by jonstokes** [comment]
@nalanbar you may be right that using the network to authorize CCW is a better way to go. I'm totally open to that. You should submit a PR and we can discuss.

**Comment by jonstokes** [comment]
@tsgsjeremy I think that there are ways we can merge this idea with the one I have in the PR.

The part on privacy could be moved wholesale into my proposal. 

The structure is also basically the same as what I'm proposing: the central authority licenses the "shooter networks", and those "shooter network" entities license individual shooters.

The enforcement part is where we differ the most. I basically want to have a few pretty harsh (but not catastrophically so) and clearly delineated penalties for loss of trust, in exchange for having relatively few ways to enter a non-trusted state.

Anyway, take a look at my PR and let's maybe have a discussion about merging these ideas with it.

**Comment by nalanbar** [comment]
@jonstokes looks like a permission error? I'll admit, I'm not super used to using git.

**Comment by tsgsjeremy** [comment]
@jonstokes I have now read the PR. (Day job has been kicking my butt.) All in all it seems we are thinking along the same lines. We do differ quite a bit on enforcement, but I'm not opposed to some compromise.

The note about the 2A was interesting. I avoided 2A issues by conceiving of a non-government entity. Again, like SSL. I envision cooperation and special agreements - like NICS checks - with government but not a scenario where government is in control. (FYI - my approach has problems too, as noted below.)

After thinking about the 2A problems, I'm not sure the government is tenable. It would require at least a re-interpretation, possibly even repeal. The same would be true for making unlicensed possession or touching a gun a felony.

By privatizing the organization and using existing legal entities for shooting networks this could be handled through contracts. Rather than making unlicensed handling a felony, the contract would stipulate that violators must surrender their firearms to their network. In a sense the firearms are treated as collateral in the deal.

I'm fine with harsh penalties, but we need to be careful not to dis-incentivize self reporting. For example, if a member or group reports another member for a violation and the group is disbanded, other groups would be reluctant to report violations. Perhaps disband the group if a member commits a violation without the groups knowledge, but not in cases where the group is pro-active.

Known Issues
- Since my proposal relies on voluntary adoption and implementation, it would need significant buy-in to be effective. 2A aside, a government solution could be mandated.
- Non-members would still be able to buy/sell/trade under the protection of the 2A, thus reducing effectiveness.

**Comment by ghost** [comment]
Getting buy-in will be tough - this is one of the more popular, well regarded forums.

https://pistol-forum.com/showthread.php?21125-Networks-of-licensed-shooters-denying-access-to-sic-bad-individuals

**Comment by tsgsjeremy** [comment]
@pepperonigun - for a government-run program I agree. I would be against that as well.

You give a couple examples - USPSA and IDPA - of private organizations that already do a limited version of what I'm proposing. (Might be worth a closer look.) They seem to be accepted by the community so I would expect a similar results.

Realistically there will always be some amount of push back because you can't please all the people all the time. I've been watching the gun debate for a long time and this is the first solution that I believe could actually work.

**Comment by ghost** [comment]
Just going to say that the number of people excluded is tiny, mostly because the number of members is also tiny in comparison to the number of gun owners probably out there...

**Comment by jonstokes** [comment]
@pepperonigun I read through that thread, and does highlight a problem with the proposal that I've encountered in one other instance: if you don't read /the entire thing/, you have no idea how it works.

In other words, if you start reading from top to bottom, then as you go you're probably flagging stuff as crazy. But in order to understand how all the parts fit together, you have to first know all the parts and then try fitting them together.

So the idea presents badly, especially to pro-gun types. This is an issue.

**Comment by jonstokes** [comment]
@tsgsjeremy the problem with a non-mandated solution is that it's hard for me to see the incentive to joining one, since I can already do whatever I want without any of the shared liability.

I agree that the 2A issue seems intractable, but IANAL, and I'd want to hear from a few of them on this.

Ultimately, I think continuing to work on this, even if it's a government mandated concept with no chance of passing anytime soon, can serve a few practical purposes.
1. It gives pro-gun people something to respond with when people ask, "so, what's your plan then... just do nothing?" Having this in your pocket is better than nothing, even if it's a long long way from being practical.
2. It can serve as a sort of backup alternative to a major ban/confiscation program.

For the guys in the pistol forum that were calling for my head, I imagine if they took a hard look at this most of them would prefer it to a major loss of rights. In fact, I personally would prefer it to the system that we have now, because it would be a lot cheaper and less hassle for me as a gun buyer and shooter, especially with regard to NFA.

**Comment by tsgsjeremy** [comment]
> For the guys in the pistol forum that were calling for my head, I imagine if they took a hard look at this most of them would prefer it to a major loss of rights.

I got the impression there was some "bandwagoning" going on.

Re: "it's hard for me to see the incentive to joining one, since I can already do whatever I want without any of the shared liability."

From my perspective, that is actually the carrot. I join the network to protect the 2A which protects my right vis-à-vis the government.

There will always be dead beats and free riders. But I refuse to let the perfect become the enemy of the good. (As a programmer I struggle with that a lot.) And like you, I think working on this despite the obstacles could yield positive results.

**Comment by jonstokes** [comment]
What about this, then. What about if we version this thing (I'll add a commit to make what's in there now 0.0.1), and start on an 0.0.2 RC that has more of the direction we seem to be going in this thread. Specifically, a shooter network regime as a replacement for the current background check system.

The "stick" would be saying that an FFL can only transfer a firearm to a licensed shooter, and the "carrot" would be the "preservation of 2A" idea, the prospect of this actually working to cut down on lone wolf terror incidents and gun violence/misuse (because gun people do actually care about that stuff), and hassle-free NFA goodies for licensed shooters (i.e. shooter control as a replacement for the current NFA trust + LEO stuff).

We'd ditch the part about "don't touch guns if you're unlicensed" (which seems pretty dubious legally anyhow) and just say "you can't buy from an FFL or have NFA goodies if you're unlicensed". That way, guys who want to meet up in a parking lot and swap cash for guns that they found on Armslist can still roll that way (if they're in a state where that's legal), and the rest of us who aren't tinfoil hatters can have a way better gun buying/owning experience.

Thoughts?

**Comment by tsgsjeremy** [comment]
Sounds good to me. I'll be running this by my gun dealer - and personal friend tomorrow. He doesn't beat around the bush so I should get some honest feedback.

**Comment by RX-79G** [comment]
How is this different than the current US system, or Illinois' FOIA card system? It still sounds like a  popularity contest that any gang member, church member or KKK member can pass because they have friends who haven't done anything to become felons.

**Comment by jonstokes** [comment]
One of the stated goals is that the system is equal to or better than the status quo, not that it solves everything.

In the case of a KKK member who passes a background check, that amounts to keeping the status quo. They'd get licensed the first time out under shooter control, just like they'd get a gun under the current system. 

HOWEVER, over time, as the Klasman's shooter networks keep being dissolved because of his fellow Klansmen getting in trouble for weapons-related crimes, the "no more than five memberships in a lifetime" rule means that he'll eventually run out of people to partner up with to join a network.

**Comment by RX-79G** [comment]
So, to apply this to the most recent Muslim related shooter, do you believe the radical Muslims are going to eventually run out of family, Mosque members and friends? I'm not super with math, but if you take the number of people that are possibly likely to be involved in mass shooting in the US Muslim, then subtract 4 people for each of them, how many US Muslims able to form networks do you have left. (Not picking on Muslims, just using the example).

I think you'll find the answer is "millions". For the 30,000 Crips (just the one gang), the number of associated people is going to be over 100,000. For a quasi-secret organization like the KKK, how many friends, associates, members and family members does that 8000 member group have to expend on networks?

What math are you seeing?

**Comment by tsgsjeremy** [comment]
That's a good point RX. I _sorta_ offered a partial answer on another thread, but you've given me more to think on. I.E. If Net A certifies Net B with 1 or 2 undiscovered bad actors, Net B could certify Net C with all bad actors. So if Net B is responsible for re-certification the defective Net C could go undiscovered until they do something really bad.

I'll have to think on that for a while. My gut reaction is that this is a structure + management issue that _should_ have a solution.

In the meantime, I talked to my FFL today. He seemed to think that straw purchases would be a big problem. It plays into the above scenario to a degree. In essence the 1 or 2 bad actors in Net B are "straw members" granting certification to others who wouldn't normally be certified.

**Comment by tsgsjeremy** [comment]
OK, I've been thinking on this and have a possible solution loosely based on protocols used to renew security clearances, with a dash of two-person-integrity.

Assuming the Net A, B, and C example above...

Rather than letting the original certifying network re-certify the sub-network, that task could be assigned randomly to another un-related network.

The FBI handles background checks for, issues and renews security clearances. The same agent is rarely if ever assigned to review the same subject. They just keep rotating them around to prevent collusion.

That led me to review the current vetting procedures for a security clearance, which are almost exactly the same as they were 20 years ago. (I only noticed 1 difference.) I think the process would be a good starting point for network/member vetting. We'll need standard guidelines that all networks must follow, because the intangible "trust" metric could vary wildly from network to network. Ex: A southern conservative and Seattle liberal might not get a warm fuzzy from each other.

**Comment by tsgsjeremy** [comment]
Whoa, just got my ass handed to me. LOL I sent FFL Steve to this site to review and he called to rant for almost an hour. Might want to change the proposal documentation to remove all the gov't references. He was so put off by the fact that it was even considered - specifically that it was the FIRST thing considered - that he has completely soured on the idea even if it is completely private and completely voluntary. Because you are trying to pass a law and take away his rights.

On the plus side, it does provide a glimpse of the real-world implications of a trust based system. He's made up his mind that he doesn't trust Jon! :) You know what they say about first impressions.

The conversation was also illustrative of the innate biases and blind spots that we all have. He talked about the dangers of a gun registry and how the gov't could subpoena the list and use it to confiscate all the guns. Similar to what happened in Australia. I asked him if he kept a record of all his sales. "Yes." What prevents the gov't from subpoenaing YOUR records and confiscate all the guns? "Congress."

Now, it may sound like I'm blowing him off or making light of his objections. NO. They are the same objections I had when I read the proposal as-is. This topic hits closer to home for him so his level of passion is higher. That's the only difference. And it should be obvious from the comments here an on other sites that these objections are nearly universal in the pro-gun camp.

**Comment by RX-79G** [comment]
I thought this was a trust network, were the validation came from personal relationships.

How are two non-related networks going to validate each other if neither has any power to investigate? I can't look up your criminal record or comment on your strange personal habits. FBI agents have the power to look at non-public records, and that is pretty much all they are doing. This is the opposite of how a trust network works.

Who in the gun control camp is for this?

**Comment by tsgsjeremy** [comment]
> I thought this was a trust network, were the validation came from personal relationships.

We're still trying to find out the right balance between set standards and personal relationships. Minimum standards would be required to provide objective measures for disqualification. And personal relationships are needed for two main reasons: 1) to apply the "I trust that guy with a gun" test, and 2) to hold people responsible for vouching for that guy.

I never said networks should follow FBI procedure, only that the FBI background check is a good starting point for validation. And I don't mean this in a bad way, but you're just wrong about how security clearance background checks work. First you fill out a very detailed form. Then they run a check for auto-disqualifiers. These are usually things admitted to on the form. But they could be a listing on the no-fly list, terror watch list, FBI most wanted list, etc. Some of that info is non-public, but I can run a NICS check to get them to do it for me. If pass, then they run a criminal records check. I can do the same thing for $20 online. (and not some fly-by night "Find your old girlfriend" outfit) They take the results and compare them to the form to see if you lied about anything or left something off that was disqualifying. If pass, then they interview you to get a sense of your behavior. Are you hiding something? Are you shifty? Are you trustworthy? Again, this is pass/fail. Then they send folks to the references you list on the form to and ask those same questions and get referrals for other people that you DIDN'T list as references. They did that 3 levels deep in my case. People called to find out why the FBI was asking questions about me.

I don't know how much of that would be required or practical. And while the system isn't fool proof - Snowden, Manning - it is pretty good at determining who is trustworthy and who isn't.

> Who in the gun control camp is for this? 

Who administers NICS again? LOL They may not be for it, but they already have to live with it.

**Comment by jonstokes** [comment]
In the docs I decided to foreground the negative stuff (for gun owners) as a way of pitching it to anti-gunners, first. Clearly that was a tactical mistake, because pro-gun people almost uniformly hate the proposal.

The second mistake is how this is positioned, i.e. as a proposal for what we should do next. In reality, I think of this more as "here is what we throw out on the table for the other side to chew on as an alternative to whatever heinous thing they're trying to get through a Dem-controlled congress."

In other words, gun people in some states have it pretty good right now, so why would they go for this if they're lucky enough to live in a place where what they have is working well for them? They won't. This has to be more of a, "what if the CA regime is suddenly on the table for the entire nation" type of thing.

**Comment by RX-79G** [comment]
As someone who can see both sides of the issue, what about your proposal would be attractive to gun control people?

I don't understand why a gun control advocate would see anything of value in gun owners validating each other instead of the DOJ doing it. While it might exclude the right people over time, the first 5 years or more would be a buying free for all.
